### ISSUE TITLE
"todo open" command to open a task on the web

### ISSUE THREAD

**Issue by jelmer** [pimutils/todoman#381]
it would be great if there was something like a "todo open" command that accessed the URL property of a vTODO (if set) and opened it, e.g. with sensible-browser.

**Comment by WhyNotHugo** [comment]
Makes sense to me. Is there any standard cli command we can call that'll reliably use to summon a browser?

I was convinced that `open` was a standard cli way of doing this, but apparently it's just [an alias](https://github.com/WhyNotHugo/dotfiles/blame/e5eeec0b4b961db075d617ec7f6998e37ee5f43a/home/.config/sh/aliases.sh#L19) I set for myself six years ago.

Maybe $BROWSER ?

**Comment by jelmer** [comment]
The webbrowser module (part of Python itself) can be used to open URLs; it obeys $BROWSER, but in the absence of that will try various other browsers.

**Comment by WhyNotHugo** [comment]
Huh. Perfect! Makes perfect sense to me. Sounds like a nice and easy enhancement!
